x-pack/test/functional/config.base.js

```diff
@@ -36,7 +36,11 @@ export default async function ({ readConfigFile }) {
     esTestCluster: {
       license: 'trial',
       from: 'snapshot',
-      serverArgs: ['path.repo=/tmp/', 'xpack.security.authc.api_key.enabled=true'],
+      serverArgs: [
+        'path.repo=/tmp/',
+        'xpack.security.authc.api_key.enabled=true',
+        'cluster.routing.allocation.disk.threshold_enabled=true', // make sure disk thresholds are enabled for UA cluster testing
```

Comment:
Isn't this being set as part of the `before()` hook in the test? Is it needed in both places?

```
        // Trigger "Low disk watermark" ES Upgrade readiness check
        await es.cluster.putSettings({
          body: {
            persistent: {
              cluster: {
                // push allocation changes to nodes quickly during tests
                info: {
                  update: { interval: '10s' },
                },
                routing: {
                  allocation: {
                    disk: {
                      threshold_enabled: true,
                      watermark: { low: '30%' },
                    },
                  },
                },
              },
            },
          },
        });
```

Reply:
Ideally it would be only in the `before` hook but ES does not reflect this change to the cluster without a restart. I added it in both to be explicit that we need this configuration. We can remove it if you feel strongly about it

Reply:
Ah, I see. Thanks for the explanation. I'm OK to leave as-is.